### ISSUE TITLE
New repo: Tanks: The Modern Age

### ISSUE THREAD

**Issue by JBardey** [BSData/catalogue-development#158]
There used to be some list builders for Tanks: The Modern Age (it's got some changes to the original Tanks: WW2 game, but has been replaced by World Of Tanks now).

[I have created a repo with the data files](https://github.com/JBardey/gf9tma). I own most of the USA, British and Soviet expansions, so these should be mostly complete, German and French are workable, but don't have any country specific upgrade cards.

There are a few things I need to clean up in my GitHub repo, but the docs for BSData aren't 100% clear.

**Comment by amis92** [comment]
@JBardey we already have both https://github.com/BSData/tanks and https://github.com/BSData/world-of-tanks-miniatures-game

It's a bit confusing which game is the replacement of which?

**Comment by JBardey** [comment]
Yeah, it's not the easiest is it? Questionable marketing decisions from Gale Force 9. 

So there are 3 games and none of the games are compatible with each other.

* **Tanks: World War 2** - This was the first game, set in World War 2, released 2016.
* **Tanks: The Modern Age** - This is the second game, set during the Cold War era, has a few additional rules, introduces helicopters. Released 2018.
* **World of Tanks: Miniatures Game** - This is the latest game and really dropped about 3 or 4 months ago. It's a tie in to the PC game, otherwise a re-release of Tanks: World War 2 with some new rules from their tournament packs for the original game. They doubled the points for easier 'balancing' but also means your 2016 Tanks: World War 2 cards are no longer valid.

https://github.com/BSData/tanks is the WW2 2016 version, https://github.com/BSData/world-of-tanks-miniatures-game is the 2019 version, I wrote all the cards I had as Battlescribe for the 2018 version for my personal use and thought I could share them with the community. It was a great excuse to tinker around with Battlescribe anyway.
